src/briefcase/integrations/android_sdk.py

```diff
@@ -1258,7 +1270,7 @@ def run(self, *arguments, quiet=False):
                 raise InvalidDeviceError("device id", self.device) from e
             raise
 
-    def install_apk(self, apk_path):
+    def install_apk(self, apk_path: SubprocessArgT):
```

Comment:
`SubprocessArgT` is `Union[str, Path]`....so, yes. But I think I see your point. I typed this based on the usage of `apk_path` with `install_apk()` instead of how the signature should be presented to devs.

I'll update it to `str | Path`.